### ISSUE TITLE
[v0.16.0rc1 backend] timeouts while trying to load /@nonexistingaccount page

### ISSUE THREAD

**Issue by Gandalf-the-Grey** [steemit/steemit.com#693]
Works fine in current setup. 
Causes trouble when used with `v0.16.0rc1` backend

**Comment by roadscape** [comment]
This is what I'm seeing when I try to load an invalid account with `v0.16.0rc1`

![image](https://cloud.githubusercontent.com/assets/5168676/20536935/1d7b0a4c-b0b9-11e6-84ca-001533f97906.png)

![image](https://cloud.githubusercontent.com/assets/5168676/20536926/1549be0e-b0b9-11e6-903c-792ff8120b4e.png)

**Comment by svk31** [comment]
Looks fine now:

![image](https://cloud.githubusercontent.com/assets/6890015/20806717/02951034-b7ca-11e6-80da-35330dc0c7df.png)

**Comment by roadscape** [comment]
Now I'm seeing the issue..

![image](https://cloud.githubusercontent.com/assets/5168676/20813543/fd822e6e-b7e3-11e6-86e0-8ecc1ac7838e.png)

```
>>> get_state "/@qqqqqqqqqqqqqqqqq"

{
  "current_route": "/@qqqqqqqqqqqqqqqqq",
  "props": {
    "id": 0,
    "head_block_number": 7216383,
    "head_block_id": "006e1cffb28df03561504d5285d1c49c208dce4d",
    "time": "2016-12-01T21:35:27",
    "current_witness": "clayop",
    "total_pow": 344152,
    "num_pow_witnesses": 110,
    "virtual_supply": "241264186.944 STEEM",
    "current_supply": "233069573.127 STEEM",
    "confidential_supply": "0.000 STEEM",
    "current_sbd_supply": "1240875.642 SBD",
    "confidential_sbd_supply": "0.000 SBD",
    "total_vesting_fund_steem": "213055323.016 STEEM",
    "total_vesting_shares": "453345448842.653935 VESTS",
    "total_reward_fund_steem": "64598.624 STEEM",
    "total_reward_shares2": "1217396382040285218419281765729",
    "sbd_interest_rate": 1000,
    "sbd_print_rate": 5366,
    "average_block_size": 1573,
    "maximum_block_size": 65536,
    "current_aslot": 7264309,
    "recent_slots_filled": "340282366920938463463374607431768211455",
    "participation_count": 128,
    "last_irreversible_block_num": 7216363,
    "max_virtual_bandwidth": "5986734968066277376",
    "current_reserve_ratio": 20000,
    "vote_regeneration_per_day": 40
  },
---- snip ----
  "content": {},
  "accounts": {
    "qqqqqqqqqqqqqqqqq": {
      "id": 0,
      "name": "",
      "owner": {
        "weight_threshold": 0,
        "account_auths": [],
        "key_auths": []
      },
      "active": {
        "weight_threshold": 0,
        "account_auths": [],
        "key_auths": []
      },
      "posting": {
        "weight_threshold": 0,
        "account_auths": [],
        "key_auths": []
      },
      "memo_key": "STM1111111111111111111111111111111114T1Anm",
      "json_metadata": "",
      "proxy": "",
      "last_owner_update": "1970-01-01T00:00:00",
      "last_account_update": "1970-01-01T00:00:00",
      "created": "1970-01-01T00:00:00",
      "mined": false,
      "owner_challenged": false,
      "active_challenged": false,
      "last_owner_proved": "1970-01-01T00:00:00",
      "last_active_proved": "1970-01-01T00:00:00",
      "recovery_account": "",
      "last_account_recovery": "1970-01-01T00:00:00",
      "reset_account": "",
      "comment_count": 0,
      "lifetime_vote_count": 0,
      "post_count": 0,
      "can_vote": false,
      "voting_power": 0,
      "last_vote_time": "1970-01-01T00:00:00",
      "balance": "0.000 STEEM",
      "savings_balance": "0.000 STEEM",
      "sbd_balance": "0.000 STEEM",
      "sbd_seconds": "0",
      "sbd_seconds_last_update": "1970-01-01T00:00:00",
      "sbd_last_interest_payment": "1970-01-01T00:00:00",
      "savings_sbd_balance": "0.000 STEEM",
      "savings_sbd_seconds": "0",
      "savings_sbd_seconds_last_update": "1970-01-01T00:00:00",
      "savings_sbd_last_interest_payment": "1970-01-01T00:00:00",
      "savings_withdraw_requests": 0,
      "vesting_shares": "0.000 STEEM",
      "vesting_withdraw_rate": "0.000 STEEM",
      "next_vesting_withdrawal": "1970-01-01T00:00:00",
      "withdrawn": 0,
      "to_withdraw": 0,
      "withdraw_routes": 0,
      "curation_rewards": 0,
      "posting_rewards": 0,
      "proxied_vsf_votes": [],
      "witnesses_voted_for": 0,
      "average_bandwidth": 0,
      "lifetime_bandwidth": 0,
      "last_bandwidth_update": "1970-01-01T00:00:00",
      "average_market_bandwidth": 0,
      "last_market_bandwidth_update": "1970-01-01T00:00:00",
      "last_post": "1970-01-01T00:00:00",
      "last_root_post": "1970-01-01T00:00:00",
      "post_bandwidth": 0,
      "vesting_balance": "0.000 STEEM",
      "reputation": 0,
      "transfer_history": [],
      "market_history": [],
      "post_history": [],
      "vote_history": [],
      "other_history": [],
      "witness_votes": [],
      "blog_category": {}
    }
  },
  "pow_queue": [],
  "witnesses": {},
  "discussion_idx": {},
  "witness_schedule": {
    "id": 0,
    "current_virtual_time": "0",
    "next_shuffle_block_num": 1,
    "current_shuffled_witnesses": "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
    "num_scheduled_witnesses": 1,
    "top19_weight": 1,
    "timeshare_weight": 5,
    "miner_weight": 1,
    "witness_pay_normalization_factor": 25,
    "median_props": {
      "account_creation_fee": "0.001 STEEM",
      "maximum_block_size": 131072,
      "sbd_interest_rate": 1000
    },
    "majority_version": "0.0.0"
  },
  "feed_price": {
    "base": "0.154 SBD",
    "quote": "1.017 STEEM"
  },
  "error": "13 N5boost16exception_detail10clone_implINS0_19error_info_injectorISt12out_of_rangeEEEE: unknown key\nunknown key: \n    {\"name\":\"qqqqqqqqqqqqqqqq\",\"what\":\"unknown key\"}\n    th_a  database.cpp:322 get_account"
}
```

1. The backend should not be returning invalid accounts in the state
2. We should be aware of the error key

**Comment by svk31** [comment]
If we need a short-term fix until it's fixed in the backend it looks like we can reject accounts with `id=0` or `name=""`

**Comment by svk31** [comment]
Ok this is happening in prod now, it's the `get_state` call in UniversalRender that times out after 60 seconds.

```
  <-- GET /@svazdazdazdazd
SteemApi exec 0 get_state ( [ '/@svazdazdazdazd' ] )
WARNING! ws connection wss://steemit.com/wspa closed:  Connection dropped by remote peer.
WebSocket: will try to reconnect in 1 sec, attempt #1
State/store error:  connection closed [no stack]
```

Besides decreasing our timout delay there's not much we can do about this in the frontend, ideally the backend would throw an error for accounts that don't exist.

**Comment by roadscape** [comment]
I believe we're waiting on https://github.com/steemit/fc/pull/6 for a fix here.

**Comment by roadscape** [comment]
This issue has been resolved with the update to 0.16.1.
